docs/manual/java/guide/cluster/PersistentEntityCassandra.md

```diff
@@ -70,6 +70,18 @@ With these properties set to `false`, if the keyspaces or tables are missing at
 
 Lagom's Cassandra support is provided by the [`akka-persistence-cassandra`](https://doc.akka.io/docs/akka-persistence-cassandra/0.100/) plugin. A full configuration reference is available in the plugin's [`reference.conf`](https://github.com/akka/akka-persistence-cassandra/blob/v0.100/core/src/main/resources/reference.conf).
 
+### Important Note Regarding `lagom-persistence-cassandra` Configuration
+
+While persistent entities store their events in a journal, the initial offset scanning for persisted events is different from [akka-persistence's configuration](https://doc.akka.io/docs/akka-persistence-cassandra/current/events-by-tag.html) such that the initial offset is *overridden* by [lagom-persistence-cassandra](https://github.com/lagom/lagom/blob/master/persistence-cassandra/core/src/main/resources/play/reference-overrides.conf):
+```conf
+cassandra-query-journal {
+  first-time-bucket = "20160225T00:00"
+
+  plugin-dispatcher = "lagom.persistence.dispatcher"
+}
```

Comment:
I'ts also important to note that once the application has one event stored the value of `first-time-bucket` can't be a timestamp _after_  the timestamp of that first event otherwise new projections will not see those initial events.

Reply:
Would you say the following is worded better?

Using Cassandra for persistent entities relies on `akka-persistence`, which on application startup, must scan from an initial offset timestamp for the first events possibly recorded for those persistent entities. In [akka-persistence configuration](https://doc.akka.io/docs/akka-persistence-cassandra/current/events-by-tag.html), the `first-time-bucket` configuration is overridden by lagom as under:
```hocon
cassandra-query-journal {
  first-time-bucket = "20160225T00:00"
}
```
Which, for newer applications first launching, can cause a not-so-insignificant delay in reading from the event journal to the latest events. It's recommended to set this `first-time-bucket` value to something closer to relevancy to reduce any delay of event-persisted entities states. Likewise, it's important that the `first-time-bucket` is not a timestamp *after* the first event, otherwise new projections may suffer from data invariance.

Reply:
@gabizou, yeah. This looks much better. 

The two last sentence would be even shorter.

Instead of: "It's recommended to set this first-time-bucket value to something closer to relevancy to reduce any delay of event-journal based projections (such as topic subscribers). Likewise, it's important that the first-time-bucket is not a timestamp after the first event, otherwise new projections may suffer from data invariance."

It could be:
"Therefore, make sure to set `first-time-bucket` to the date of the first deployment of your application when the journal was completely empty."

Maybe talking about relevancy or data invariance can be a distraction. The message we want to pass is that users should set it to the date of first deployment. That's all.

Reply:
Replaced the comments in the PR with the suggestions